.golangci.yml

```diff
@@ -72,6 +72,34 @@ linters-settings:
       # Controller Runtime
       - pkg: sigs.k8s.io/controller-runtime
         alias: ctrl
+      # CAPI
+      - pkg: sigs.k8s.io/cluster-api/api/v1alpha3
+        alias: clusterv1alpha3
+      - pkg: sigs.k8s.io/cluster-api/api/v1alpha4
+        alias: clusterv1alpha4
+      - pkg: sigs.k8s.io/cluster-api/api/v1beta1
+        alias: clusterv1
+      # CAPI exp
+      - pkg: sigs.k8s.io/cluster-api/exp/api/v1alpha3
+        alias: expv1alpha3
```

Comment:
Maybe it should be `expclusterv1alpha3` to be consistent with the above?

Reply:
The CAPI rules for `importas` were copied from CAPI's golangci-lint configuration. I think we should just use them as-is to make it easier to move between codebases. I notice that's what CAPA did as well.

Reply:
Well, some say you should always question things and there is always room for the improvement... :smile: But perhaps less optimal consistency is better than better inconsistency, so I guess that's OK. Thanks for explaining! Feel free to mark as resolved.